website/scripts/bak/build-all/index.ts

```diff
@@ -0,0 +1,27 @@
+import Listr from 'listr';
```

Comment:
Can all this `bak` directory be removed ?

Reply:
no, it's from [docusaurus](https://docusaurus.io/docs)

Reply:
it's has code used to generate pages and menues